ceno_zkvm/src/bin/e2e.rs

```diff
@@ -142,9 +151,25 @@ fn main() {
         Checkpoint::PrepSanityCheck,
     );
 
-    let (mut zkvm_proof, verifier) = state.expect("PrepSanityCheck should yield state.");
+    let (zkvm_proof, vk) = state.expect("PrepSanityCheck should yield state.");
+
+    let proof_bytes = bincode::serialize(&zkvm_proof).unwrap();
```

Comment:
Aware we have two serialization framework: bincode/rykv both serve similar purpose
I guess this is to align with openvm as we need to load proof in recursion?

Reply:
bincode is compatible with serde traits.  While rykv you need to derive its trait on every structs.. Unless we want extreme performance, we usually dont want to use rykv, the API is much more unfriendly.